css/main.css

```diff
@@ -1165,6 +1165,19 @@ li.tg-rss a{background:#ff6600;}
 	width:38px;
 	line-height:38px;
 }
+
+@media screen and (max-width: 767px) {
```

Comment:
@amoghj8 the thing what @CuriousLearner is trying to say I believe is you can set either one (wither max-width or min-width) and set the other as the default font-size. So something like by default it is 2em, but if media screen width below 767px (i.e max-width:767px) then `font-size:1.5em`. Keeping both max-width and min-width is weird.